src/pages/home-page/index.tsx

```diff
@@ -26,6 +26,7 @@ function HomePage() {
     // 'select'ing is a way to get stuff from the redux store
     // useDispatch and useSelector
     dispatch(addItemToList(item));
+    localStorage.setItem(id.toString(), JSON.stringify(item))
```

Comment:
You should not be writing each task to a separate key, instead a better approach would be to set the entire list under a storage key named `list` and stringifying the array and passing that in.